infinispan-remote/src/test/resources/hotrodclient-minimal.properties

```diff
@@ -0,0 +1,7 @@
+# empty HotRod configuration file
```

Comment:
Very nice!

Could we also test that everything works when there is no hotrod properties file at all ? (if we don't already do?)

Reply:
Currently it wouldn't work without a hotrod properties file. Are we sure to allow that? I think that at least a --host:port-- combination is required, in a real, non localhost, cases. Isn't it?

Reply:
But itsn't it silly to require to have an empty file?

I would expect that when the user does not specify a configuration resource name / filename, we will apply the defaults.

Reply:
It is, I agree. I will change it.